com.arc.embeddedcdt/src/com/arc/embeddedcdt/common/ArcGdbServer.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * This program and the accompanying materials
+ * are made available under the terms of the Common Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/cpl-v10.html
+ * 
+ * Contributors:
+ *     Synopsys, Inc. - ARC GNU Toolchain support
+ *******************************************************************************/
+
+package com.arc.embeddedcdt.common;
+
+public enum ArcGdbServer {
+    JTAG_OPENOCD("JTAG via OpenOCD"),
+    JTAG_ASHLING("JTAG via Opella-XD"),
+    NSIM("nSIM"),
+    GENERIC_GDBSERVER("Generic gdbserver");
+
+    private final String string;
+    public static final ArcGdbServer DEFAULT_GDB_SERVER = JTAG_OPENOCD;
+    public static final String DEFAULT_GDB_SERVER_STRING = DEFAULT_GDB_SERVER.toString();
```

Comment:
Do we really need a separate string for this? I understand that this allows to avoid extra calls to virtual method `toString()`, but that looks like a microoptimization in grand scheme of things... I tend to think that it would be more readable to use `DEFAULT_GDB_SERVER.toString()` instead of this field in all cases.